src/capture/middlewares/protobuf_middleware/protobuf_middleware.h

```diff
@@ -13,13 +13,24 @@
 
 #include "../../middleware.h"
 
+/**
+ * @brief pipe the serialised protobuf tuple packets
+ *
+ * @param path[in] The pipe file path
+ * @param fd[in] The pipe file descriptor
```

Comment:
This variable may be changed by the function.

```suggestion
 * @param fd[in,out] The pipe file descriptor.
 * If `0`, this will be set to a new fd pointing to @p path.
```